_posts/2017-03-23-omero-5.2.8.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: [SECURITY] Release of OMERO 5.2.8
+intro-blurb: The OME team is pleased to announce the release of OMERO 5.2.8, a security update which we recommend all sysadmins upgrade to as soon as possible.
+---
+Today we are releasing OMERO 5.2.8. This is a security release to prevent users from deleting or editing official scripts and from accessing other people's data by adjusting the file path of their OriginalFiles so it points to other people's data in the repository.
+
+Full details of the issues are available on:
+
+https://www.openmicroscopy.org/site/products/omero/secvuln/2017-SV1-filename
+https://www.openmicroscopy.org/site/products/omero/secvuln/2017-SV2-edit-rw
+https://www.openmicroscopy.org/site/products/omero/secvuln/2017-SV3-delete-script
+
+This release does not upgrade the version of Bio-Formats which OMERO uses.
+
+Upgrade information is at http://www.openmicroscopy.org/site/support/omero5.2/sysadmins/server-upgrade.html
```

Comment:
Upgrade information is on the server upgrade page -> link text as per 5.3.0 please